src/components/how-we-work/HowWeWorkWrapper/index.jsx

```diff
@@ -20,29 +23,38 @@ const videoLink = 'https://www.youtube.com/embed/MjS5OVA7ugo?autoplay=1';
 function HowWeWorkWrapper() {
   const { isLoggedIn } = useUser();
   const navigate = useNavigate();
-
-  useEffectOnce(() => {
-    if (!isLoggedIn) navigate('/sign-up');
-  });
-
   return (
-    <HowWeWorkContainer>
-      <VideoContainer>
-        <TextContainer>
-          <h2>How We Work</h2>
-          <p>{howWeWorkDescription}</p>
-        </TextContainer>
-        <div>
-          <StyledIframe
-            src={videoLink}
-            title="YouTube video player"
-            frameborder="0"
-            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-            allowfullscreen
-          />
-        </div>
-      </VideoContainer>
-    </HowWeWorkContainer>
+    <>
+      {' '}
+      <HowWeWorkContainer>
+        <VideoContainer>
+          <TextContainer>
+            <h2>How We Work</h2>
```

Comment:
```suggestion
            <h1>How We Work</h1>
```
always use h1 for page titles